src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -145,15 +145,10 @@ internal void CreateLuceneIndexWriter()
             }
             catch (LockObtainFailedException ex)
             {
-                try
-                {
-                    writer = new IndexWriter(new RAMDirectory(), indexConfig);
```

Comment:
This is no longer needed because after the performance fix, even user launches two Dynamo sessions at the same time, they wont be able to block each other with the current change.